docs/pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
I dunno why this keeps flip flopping

Reply:
I had a similar thing in https://github.com/latticexyz/mud/pull/1061 on pnpm 8.6.7, assume it's just some of our lockfiles being outdated since pnpm rolled back their version to 6.0